actions/bellsoft-liberica-dependency/main.go

```diff
@@ -169,14 +169,20 @@ func main() {
 	outputs.Write()
 }
 
-func determineNikVersion(r Release, additionalOutputs actions.Outputs) string {
+func determineNikVersion(r Release, additionalOutputs actions.Outputs, isSource bool) string {
 	key := ""
-	if v, err := actions.NormalizeVersion(r.Components[0].Version); err != nil {
+	var componentKey int
+	if isSource {
+		componentKey = 2
+	} else {
+		componentKey = 0
+	}
```

Comment:
```suggestion
	var componentKey int
	if isSource {
		componentKey = 2
	} else {
		componentKey = 0
	}
```

I think this ^^ is the same as:

```
        var componentKey int
	if isSource {
		componentKey = 2
	}
```

because go initializes variables to their default value, which would be 0 here.

If you don't like relying on that, which I'm not a fan of, you can make it explicit.

```
        componentKey := 0
	if isSource {
		componentKey = 2
	}
```

I don't fully understand it, but Go styles guides tell you not to use `else` blocks, especially for initialization like this.